processor/abci.go

```diff
@@ -1244,9 +1244,12 @@ func (app *App) CheckSubmitOracleData(_ context.Context, tx abci.Tx) error {
 	}
 
 	pubKey := crypto.NewPublicKey(tx.PubKeyHex(), tx.PubKey())
-	_, err := app.oracles.Adaptors.Normalise(pubKey, *data)
+	oracleData, err := app.oracles.Adaptors.Normalise(pubKey, *data)
+	if err != nil {
+		return err
```

Comment:
Except if the underlaying errror is very clear for what's happening, could you wrap this error with `fmt.Errof()` ?

Something that can help understand *where* things went wrong :)

Reply:
Yes,
I did some optimizations on this here https://github.com/vegaprotocol/vega/pull/5534/commits/587bd850dcca9027fd5422ef1cf618ff087a449d#diff-43cb75f5e3d4bfc8b8fc4540853a32c32346ec1fb151bbaed720d7bf07a4a139R59